doc/cross_validation.rst

```diff
@@ -0,0 +1,40 @@
+Time Gap Split Cross Validation
+===============================
+
+TimeGapSplit provides 3 parameters to really reproduce your production implementation in your cross-validation schema.
```

Comment:
I added a line above.

Reply:
It might be better to explain "why" you would want to have a gap between training and validation. It might be because you want to be very strict in terms of robustness and you would prefer this to be handled on the cross validation level.

Reply:
Ok, must be more clear, I found a good explanation, thanks for insisting. Tell me if those 2 points above are ok, then we merge. @koaning